### PR TITLE
 add scale bars to figure5

### PR DESCRIPTION
**Pull request by MikeLippincott**
This PR adds scale bars to the images in figure5 :D

### CODE REVIEW

File: figures/5.figure5/scripts/2.add_scale_bars.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    print(image_path)
+    # open the image
+    image = cv2.imread(image_path, cv2.IMREAD_UNCHANGED)
+    # image to array
+    image_array = np.array(image)
+    resolution = 0.1550  # pixels per micrometer
+    # get the image size
+    image_size = image_array.shape
+    if image_array.shape[1] <= 100:
+        scale_bar_length = 5  # um
```
Comment (jenna-tomkinson) [inline figures/5.figure5/scripts/2.add_scale_bars.py:52]: How did you identify these? Just for future reference and implementation, how would you find these values?

File: figures/5.figure5/figures/figure5.png
Comment (jenna-tomkinson) [inline figures/5.figure5/figures/figure5.png:1]: Just one FYI comment, panel A legend is cut off on the right side (`pyroptosis` is cut off).

**Review by jenna-tomkinson — APPROVED** [review]
Looks great! I made one small comment for you to address on the figure and I asked one question for the cool code where you add scale bars 😄 Once addressed, feel free to merge!

File: figures/5.figure5/scripts/2.add_scale_bars.py (hunk 1 above)
Comment (MikeLippincott) [reply figures/5.figure5/scripts/2.add_scale_bars.py:52]: this is the crop size that I set. So for larger cells if the crops are set to say 250 pixel crops then change the value. :)

**Pull request merged**